packages/api/es/search.js

```diff
@@ -42,7 +42,10 @@ const getLocalEsHost = () => {
 
 const esTestConfig = () => ({
   node: getLocalEsHost(),
-  requestTimeout: 5000
+  requestTimeout: 5000,
+  ssl: {
```

Comment:
Do we want this committed? I _think_ it's fine since it's just the test config?

Reply:
I want it committed and it is only the test config, but if there's real reason not to I can stash it and just apply when I want to use it.  I think it's super handy to work against the AWS Cluster locally through a tunnel.